lib/tokenize.cpp

```diff
@@ -3849,7 +3849,7 @@ void Tokenizer::createLinks2()
                 type.pop();
         } else if (token->str() == "<" &&
                    ((token->previous() && token->previous()->isName() && !token->previous()->varId()) ||
-                    Token::simpleMatch(token->next(), ">"))) {
+                    Token::Match(token->next(), ">|>>"))) {
```

Comment:
Would this link < to >>?  Shouldn't >> be broken up into 2 tokens?

Reply:
yes that is how we do it as far as I remember.

Reply:
This is only matching so it can push `<` onto the stack. The linking happens later when it encounters a `>` or  `>>`, which I dont change.